app/src/main/kotlin/com/twain/interprep/data/dao/NoteDAO.kt

```diff
@@ -37,6 +37,9 @@ interface NoteDAO {
     @Query("SELECT * FROM interview")
     fun getAllInterviewNoteMap(): Flow<List<InterviewWithNotes>>
 
+    @Query("DELETE FROM note where interviewId = :interviewId")
+    fun deleteNotesForInterview(interviewId: Int)
+
     data class InterviewWithNotes(
```

Comment:
Is this data class now not being used after this introduction of Foreign Key and deleting Notes by CASCADE ?
If this is not used anywhere, can we delete this class?

Reply:
This is used by NotesViewModel to read Notes for Interview.